clients/cobol-lsp-vscode-extension/src/services/Settings.ts

```diff
@@ -394,4 +392,18 @@ export class SettingsService {
       programFile,
     );
   }
+  public static prepareLocalSearchFolders(
+    paths: string[],
+    wsFolders: string[],
+  ): string[] {
+    paths
+      .filter((e) => !path.isAbsolute(e))
+      .forEach((localPath) => {
+        wsFolders.forEach((wsFolder) => {
+          paths.push(path.join(wsFolder, localPath));
+        });
+        paths.splice(paths.indexOf(localPath), 1);
+      });
+    return paths;
```

Comment:
I guess this works, but wouldn't something like this be easier to understand?
```suggestion
    const result = []
    for (const p of paths) {
        if(path.isAbsolute(p)) result.push(p);
        else wsFolders.forEach((wsFolder) => {
            result.push(path.join(wsFolder, p));
        });      
    }
    return result;
```